src/main/scala/com/gu/okhttp/SharedClient.scala

```diff
@@ -0,0 +1,24 @@
+package com.gu.okhttp
+
+import okhttp3.{ OkHttpClient, Response }
+import scala.util.{ Failure, Success, Try }
+
+case class ApiException(message: String) extends Throwable(message: String)
+
+object SharedClient {
```

Comment:
https://square.github.io/okhttp/4.x/okhttp/okhttp3/-ok-http-client/#okhttpclients-should-be-shared